OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -745,5 +747,68 @@ void MakeMouseFocusSettingsLive()
 			else
 				Game.Renderer.ReleaseWindowMouseFocus();
 		}
+
+		void InitHotkeyRemapDialog(Widget panel)
+		{
+			panel.Get<LabelWidget>("HOTKEY_LABEL").GetText = () => selectedHotkeyDefinition.Description + ":";
+
+			var duplicateHotkeyNotice = panel.Get<LabelWidget>("DUPLICATE_NOTICE");
+			duplicateHotkeyNotice.TextColor = ChromeMetrics.Get<Color>("NoticeErrorColor");
+			duplicateHotkeyNotice.IsVisible = () => !isHotkeyValid;
+			duplicateHotkeyNotice.GetText = () =>
+			{
+				return (duplicateHotkeyDefinition != null) ? duplicateHotkeyNotice.Text.F(duplicateHotkeyDefinition.Description) : duplicateHotkeyNotice.Text;
+			};
+
+			var defaultHotkeyNotice = panel.Get<LabelWidget>("DEFAULT_NOTICE");
+			defaultHotkeyNotice.TextColor = ChromeMetrics.Get<Color>("NoticeInfoColor");
+			defaultHotkeyNotice.IsVisible = () => isHotkeyValid && isHotkeyDefault;
```

Comment:
I guess this can be removed as it is a bit redundant. There are notices for duplicate and custom hotkey, so not displaying any notice means that this is the default.